src/client/pages/Forever/components/RedeemCode.tsx

```diff
@@ -134,6 +130,10 @@ export const RedeemCode: React.FC<RedeemCodeProps> = ({
   const [charIndex, setCharIndex] = useState(0)
   const [printCodeTimeout, setPrintCodeTimeout] = useState<number | null>(null)
 
+  const codeSchema = Yup.object({
+    code: Yup.string().required(textKeys.FOREVER_CODE_ERROR()),
```

Comment:
Ah, YES! You're so very right! So the problem was _only_ the missing translations in Lokalise and not the way they were used. Didn't look closely enough at the implementation 😳  

Question though - where is the best place to run them? To me the way it was done before was obviously confusing (lol), but that might of course just be me.

I guess if you run it on line 134 you would just have ` errors={errors.code ? errors.code : ''}` on line 196.

Reply:
What do you think makes most sense? @palmenhq

Reply:
I think it makes more sense with the previous one in a way, because we don’t need to create the yup schema on every render. Optionally if we put it in a memo but I like it more like it was previously.

Reply:
OK, sure, that makes sense 👍  I will change it!